{
  "modified": "2025-07-31T19:38:35Z",
  "published": "2025-07-31T19:38:35Z",
  "schema_version": "1.5.0",
  "id": "",
  "aliases": [
    "MAL-2025-4869",
    "GHSA-3r3m-hqgf-4wx6",
    "SNYK-JS-PYSERIAL-10338496"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "npm",
        "name": "pyserial"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-JS-PYSERIAL-10338496"
    },
    {
      "type": "PACKAGE",
      "url": "https://npmjs.com/package/pyserial"
    },
    {
      "type": "ADVISORY",
      "url": "https://github.com/advisories/GHSA-3r3m-hqgf-4wx6"
    },
    {
      "type": "ADVISORY",
      "url": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/npm/pyserial/MAL-2025-4869.json"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "16c59ab63d35d03cabbdc2ff17b62cbd0e2f232dd90740ba9fa4cc4b7309a336",
        "import_time": "2025-08-01T10:41:40.644593592Z",
        "id": "RLUA-2025-03996",
        "modified_time": "2025-07-31T19:38:35Z"
      }
    ]
  }
}
